### ISSUE TITLE
type the correct password but sometimes I can't login, sometimes I can.

### ISSUE THREAD

**Issue by lz220011** [phpmyadmin/phpmyadmin#12097]
### Steps to reproduce
1. we install phpmyadmin and use cookie auth
2. go to index page 
3. type correct username and password
### Expected behaviour
Tell us what should happen

login and jump to operation page
OR 
return error message

### Actual behaviour
Tell us what happens instead

still in index page did not jump to operation page.

### Server configuration
**Operating system**:
cent os 7
**Web server:**
php-fpm
**Database:**
mysql
**PHP version:**
5.5
**phpMyAdmin version:**
4.4
### Client configuration
**Browser:**
chrome
liebao
**Operating system:**
windows 7

**Comment by ibennetch** [comment]
phpMyAdmin 4.4 is no longer supported, I suggest upgrading if your MySQL version allows you to do so.

Any hints in the php-fpm, webserver or MySQL logs? How many login attempts fail before you're able to connect, to give us an approximate idea of how often the problem occurs? Do you ever experience trouble logging in from the command-line client? Is your user account rate limited in some way, for instance MAX QUERIES PER HOUR or MAX CONNECTIONS PER HOUR?

Try clearing your browser cache and removing the phpMyAdmin cookies (which start with "pma") and see if that helps. Does the problem also occur if using auth_type http?

**Comment by lz220011** [comment]
Thanks for your advice. We found two possible causes for this problem. 
First, Load Balance, wrong LB make the same user's requests to different servers, the later server can't read the session info.
Second, Liebao browser‘s own errors will lead to phpMyAdmin call error_report. We set SendErrorReports = never in config.inc.php .
We fixed the above problems in phpMyAdmin4.5.3.1,and it works well.
Thanks a lot.
